app/web/package.json

```diff
@@ -68,18 +66,13 @@
   },
   "devDependencies": {
     "@babel/core": "^7.26.0",
-    "@storybook/addon-actions": "^6.5.10",
-    "@storybook/addon-essentials": "^6.5.10",
-    "@storybook/addon-links": "^6.5.10",
-    "@storybook/builder-webpack5": "^6.5.10",
-    "@storybook/manager-webpack5": "^6.5.10",
-    "@storybook/react": "^6.5.10",
     "@testing-library/dom": "^10.4.0",
     "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.1.0",
     "@testing-library/user-event": "^14.5.2",
     "@types/css-mediaquery": "^0.1.4",
     "@types/geojson": "^7946.0.10",
+    "@types/glob": "^8.1.0",
```

Comment:
Oh interesting what happened here?

Reply:
I think it was being loaded as a transitive dep of some storybook package!